Makefile

```diff
@@ -0,0 +1,83 @@
+clean: clean-build clean-py clean-test
+
+clean-build:
+	rm -fr build/
+	rm -fr dist/
+	rm -fr .eggs/
+	find . -name '*.egg-info' -exec rm -fr {} +
+	find . -name '*.egg' -exec rm -f {} +
+
+clean-py:
+	find . -name '*.pyc' -exec rm -f {} +
+	find . -name '*.pyo' -exec rm -f {} +
+	find . -name '__pycache__' -exec rm -fr {} +	
+
+clean-test:
+	find . -name '.pytest_cache' -exec rm -fr {} +
+	find . -name '.coverage' -exec rm -fr {} +
+	find . -name 'htmlcov' -exec rm -fr {} +
+	find . -name '.cache' -exec rm -fr {} +
+
+format:
+	./lambdas/venv/bin/python3 -m black . &&\
+	ruff check . --fix
+	./lambdas/venv/bin/python3 -m isort lambdas/handlers/. 
+	./lambdas/venv/bin/python3 -m isort lambdas/models/. 
+	./lambdas/venv/bin/python3 -m isort lambdas/utils/. 
+	./lambdas/venv/bin/python3 -m isort lambdas/services/. 
+	./lambdas/venv/bin/python3 -m isort lambdas/tests/. 
+
+test-unit:
+	cd ./lambdas && ./venv/bin/python3 -m pytest tests/
+
+test-unit-collect:
+	cd ./lambdas && ./venv/bin/python3 -m pytest tests/ --collect-only
+
+env:
+	rm -rf lambdas/venv || true
+	python3 -m venv ./lambdas/venv
+	./lambdas/venv/bin/pip3 install --upgrade pip
+	./lambdas/venv/bin/pip3 install -r lambdas/requirements.txt
+	./lambdas/venv/bin/pip3 install -r lambdas/requirements-test.txt
+
+zip:
```

Comment:
Looks good but we may need a chat on how we deploy all lambdas at once vs individually.